### DISCUSSION TITLE
Listing() no results on production

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#10656] [Q&A / Support]
Hi, I'm testing a Pimcore X setup on Hostinger Shared Hosting.

So far it runs pretty good. I've created a product data object class. Selecting a single product works fine, but loading a list usting Listing() returns an empty array.

This happens only on production, data loading works fine on the local dev environment.

```
$product = Product::getById(11);

$listing = new Listing();
$products = $listing->load();

dump($product); // works fine local + production
dump($products); // works on local, not on production
```

Does anyone have an idea what could cause this issue?

Thanks!

**Comment by NiklasBr** [comment]
It's not clear which `use` statements you have, so I have to first ask if `Listing()` is actually the same as `Product\Listing()`, also is `$product` published? [By default a Listing will only fetch published objects](https://github.com/pimcore/pimcore/blob/10.x/models/DataObject/Listing.php#L68-L80).

**Reply by valentin-** [reply]
There was a problem with the database synced from local to master, it's working now. Thanks.
